packages/backend/src/exploregens.ts

```diff
@@ -161,12 +155,16 @@ export class ExploreGens {
     return vscode.commands.executeCommand("yeomanUI._notifyGeneratorsChange");
   }
 
+  private setStatusBarMessage(message: string) {
```

Comment:
```suggestion
  private setStatusBarMessage(message: string): Thenabe<string> {
```

Reply:
It is not Thenable, but Disposable
Done